app/src/main/kotlin/org/cru/godtools/ui/languages/LanguageName.kt

```diff
@@ -49,16 +50,25 @@ internal fun LanguageName(language: Language, modifier: Modifier = Modifier) {
 private const val LANGUAGE_NAME_GAP = "[gap]"
 
 @Composable
-private fun LanguageName(displayName: String, nativeName: String, modifier: Modifier = Modifier) {
+private fun LanguageName(
+    displayName: String,
+    nativeName: String,
+    modifier: Modifier = Modifier,
+    isForcedName: Boolean = false,
```

Comment:
I would rather see this be updated to make the secondary name optional, instead of passing an additional boolean in.

Reply:
maybe updated to be something like:
```
private fun LanguageName(
    name: String,
    modifier: Modifier = Modifier,
    secondaryName: String? = null
)
```